staging/src/k8s.io/apiserver/pkg/authentication/group/group_adder.go

```diff
@@ -31,6 +31,11 @@ type GroupAdder struct {
 	Groups []string
 }
 
+// AuthenticatorID implements the AuthenticatorID of the authenticator.Request interface.
+func (g *GroupAdder) AuthenticatorID() string {
```

Comment:
Could this be done by embedding authenticator.Request in GroupAdder?
If it get noisy, ok to do this way too